qutebrowser/utils/utils.py

```diff
@@ -42,6 +42,10 @@
 from PyQt5.QtCore import QUrl, QVersionNumber
 from PyQt5.QtGui import QClipboard, QDesktopServices
 from PyQt5.QtWidgets import QApplication
+if sys.version_info >= (3,9):
+    import importlib.resources as importlib_resources
+else:
+import importlib_resources
```

Comment:
What. How.

Reply:
[Not your first PR](https://github.com/qutebrowser/qutebrowser/pull/5768/files#diff-a3f2f10f8adb634b3c7e2c6e18e468f4fea27185be7105f840d620783667a323R233) with a syntax error FWIW :wink: